packages/render-fidelity-tools/test/config.json

```diff
@@ -404,6 +404,17 @@
         "dspbr-pt"
       ]
     },
+    {
+      "name": "khronos-MaterialsVariantsShoe",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/MaterialsVariantsShoe/glTF/MaterialsVariantsShoe.gltf",
+      "orbit": {
+        "theta":35,
+        "phi": 49
```

Comment:
Let's reduce radius here from default so that it fills the viewport better. No need for perfection, I just want to make use of the pixels we have.